projects/demo/src/app/demo/pages/expand-page/expander-page.component.html

```diff
@@ -52,6 +66,14 @@ <h3>Inputs</h3>
             <dt>contentLoaded</dt>
             <dd>Any value marking expander's content as loaded or updated so the expand can update its body height.</dd>
         </dl>
+        <dl>
+            <dt>card</dt>
+            <dd>By default card expander will be collapsed and can only be expanded by the expander button</dd>
```

Comment:
I feel like you should update the description on the top of the page to present the two options. Saying something like:
```
By default, expander are displayed like in a list: the toggle button being on the left. The <code>card</code> option allows to display a card expander with the toggle button on the right.
```

```suggestion
            <dd>Card option allows to change the expander look and feel by displaying toggle button on the right and any <code>pa-expander-header-side-block</code> on the left. When this option is set, the card expander is collapsed by default and can only be expanded by the expander button</dd>
```